packages/cds-odata-annotation-converter/src/transforms/annotation/handlers/annotation.ts

```diff
@@ -20,7 +20,7 @@ export const annotationHandler: NodeHandler<Annotation> = {
     type: ANNOTATION_TYPE,
     convert,
     getChildren(state: VisitorState, annotation: Annotation): AnnotationNode[] {
-        if (!annotation.term.value.endsWith(ReservedProperties.Type) && annotation.value) {
+        if (!annotation.term.value.includes(ReservedProperties.Type) && annotation.value) {
```

Comment:
This would also include false positive case like `Chart.My$Type`. We should check if there are any segments with the .`$Type`. Please also add a case for this.